docs/eventing/broker/kafka-broker.md

```diff
@@ -0,0 +1,112 @@
+---
+title: "Apache Kafka Broker"
+weight: 30
+type: "docs"
+---
+
+This page shows how to install and configure the 
+[Kafka Broker](https://github.com/knative-sandbox/eventing-kafka-broker).
+
+This guide assumes a 
+[Knative Eventing installation](./../../install/any-kubernetes-cluster.md#installing-the-eventing-component).
+
+## Installation
+
+<!-- TODO Where does the config for populating the following live? -->
+```bash
+kubectl apply --filename {{< artifact repo="eventing-kafka-broker" file="eventing-kafka-broker.yaml" >}}
+```
+## Create a Kafka Broker
+
+A Kafka Broker object looks like this:
+
+```yaml
+apiVersion: eventing.knative.dev/v1
+kind: Broker
+metadata:
+  annotations:
+    # case-sensitive
+    eventing.knative.dev/broker.class: kafka 
+  name: default
+  namespace: default
+spec:
+  # Configuration specific to this broker.
+  config:
+    apiVersion: v1
+    kind: ConfigMap
+    name: kafka-broker-config
+    namespace: knative-eventing
+  # Optional dead letter sink, you can specify either:
+  #  - deadLetterSink.ref, which is a reference to a Callable
```

Comment:
I think you should not spend time here explaining how to configure the deadLetterSink, that's something that should be covered in the broker docs itself

Reply:
No, strong opinion, I thought it was nice to have a full Broker definition for this particular implementation.